src/Pages/Login/Login.js

```diff
@@ -0,0 +1,248 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import { KAKAO_LOGIN_API } from '../../config';
+// import './Login.scss';
+
+class Login extends Component {
+  kakaoLoginHandler = () => {
+    window.Kakao.Auth.login({
+      success: (auth) => {
+        console.log(auth);
+        fetch(KAKAO_LOGIN_API, {
+          method: 'POST',
+          headers: {
+            Authorization: auth.access_token,
+          },
+        })
+          .then((res) => res.json())
+          .then((res) => {
+            if (res.token) {
+              localStorage.setItem('Log_Token', res.token);
+              console.log('로그인 되었습니다.');
+            } else {
+              console.log('다시 확인해 주세요.');
+            }
+          });
+      },
+      fail: function (err) {
+        console.log('a');
+        console.log('에러', err);
+      },
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <Article>
+          <LoginBox>
+            <LoginForm>
+              <LoginInner>
+                <HEAD>로그인</HEAD>
+                <InputBox>
+                  <IdText>
+                    회원 아이디/스카이패스 회원번호
+                    <Required>필수 입력</Required>
+                  </IdText>
+                  <IdInput type="text" />
+                  <PwText>
+                    비밀번호
+                    <Required>필수 입력</Required>
+                  </PwText>
+                  <PwInput type="text" />
+                </InputBox>
+
+                <BtnBox>
+                  <LoginBtn>로그인</LoginBtn>
+                  <SnsText>
+                    <SnsTitle>SNS 로그인</SnsTitle>
+                  </SnsText>
+                  <LoginBtn
+                    type="button"
+                    kakao
+                    onClick={this.kakaoLoginHandler}
+                  >
+                    &nbsp;카카오 계정으로 로그인
+                  </LoginBtn>
+                </BtnBox>
+              </LoginInner>
+              <BtnSignup href="/signup">회원가입</BtnSignup>
+            </LoginForm>
+          </LoginBox>
+        </Article>
+      </div>
+    );
+  }
+}
+
+const Article = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #c1dcf9;
+  height: 100vh;
+`;
+
+const LoginBox = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 1250px;
+  height: 80%;
+  margin: 0 600px 0 600px;
```

Comment:
```suggestion
  margin: 0 600px;
```
단축 속성 사용해줄 수 있을 것 같고요, 그런데 일반적으로 마진 600px 같이 크게 주지 않아서 같이 살펴보면 좋을 거 같습니다

Reply:
불필요하여 제거하였습니다!